app/controllers/registration_commands_controller.rb

```diff
@@ -44,6 +44,11 @@ def registration_params
     end
   end
 
+  def host_groups
+    Hostgroup.authorized(:view_hostgroups)
+             .map { |hg| hg.as_json.merge({'inherited_operatingsystem_id': hg.os&.id}) }
```

Comment:
This can be ` map { |hg| hg.as_json.merge({'inherited_operatingsystem_id': hg.inherited_operatingsystem_id }) }`
And because to_json allows `:methods` it should be enough to do:
```suggestion
             .map { |hg| hg.as_json(methods: :inherited_operatingsystem_id) }
```